### PR TITLE
release MathComp-Analysis 0.5.3

### PR DESCRIPTION
**Pull request by affeldt-aist**

### CODE REVIEW

File: released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.14" & < "8.16~") | (= "dev") }
```
Comment (palmskog) [inline released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam:17]:
```suggestion
  "coq" { (>= "8.14" & < "8.17~") | (= "dev") }
```
To support the soon-to-be-released Coq 8.16.0.

**Comment by palmskog** [comment]
cc: @MSoegtropIMC

File: released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam (hunk 1 above)
Comment (affeldt-aist) [reply released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam]: I haven't tested it yet though.

Comment (palmskog) [reply released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam]: @affeldt-aist but do you mind that we have this for now to *enable* testing? If there is some problem, I can edit the package again.

Comment (affeldt-aist) [reply released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam]: I do not mind.

Comment (affeldt-aist) [reply released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.3/opam]: FYI, it appears that an installation with Coq 8.16+rc1, OCaml 4.14.0, HB 1.3.0, and dev versions of MathComp succeeded. @gares

**Pull request merged**